ntcp_http.py

```diff
@@ -0,0 +1,65 @@
+from amaranth.lib.wiring import connect, Component, In, Out
+from amaranth.lib import stream
+from amaranth import Module
+from not_tcp.not_tcp import StreamStop
+from http_server.simple_led_http import SimpleLedHttp
+
+
+class NtcpHttpServer(Component):
+    """
+    A serial-to-HTTP server, suitable for synthesis.
+    """
+
+    tx: Out(stream.Signature(8))
+    rx: In(stream.Signature(8))
+
+    led: Out(3)
```

Comment:
Is RGB on Fomu just boolean? I guess that makes sense. Maybe a PWM driver is a stretch goal.

Reply:
No, this is just me being dumb. The LED driver module is actually fairly fancy, it has a "breathe" setting, maybe a blink setting? and some other things.

And, yeah, has 8 bits / channel. Fixed.